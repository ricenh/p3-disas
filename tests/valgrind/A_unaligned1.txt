==1602934== Memcheck, a memory error detector
==1602934== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1602934== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1602934== Command: ../y86 -a -d inputs/unaligned.o
==1602934== 
01 00 18 01 10 00 03 00  00 00 00 00 45 4c 46 00
Mini-ELF version 1
Entry point 0x118
There are 3 program headers, starting at offset 16 (0x10)
There is no symbol table present
There is no string table present
 Segment   Offset    Size      VirtAddr  Type      Flags
  00       0x004c    0x0009    0x0000    CODE      R X
  01       0x0055    0x0002    0x0118    CODE      R X
  02       0x0057    0x0008    0x013a    DATA      RW 
Contents of memory from 0000 to 0009:
  0000  70 18 01 00 00 00 00 00  00
Contents of memory from 0118 to 011a:
  0110                           10 00
Contents of memory from 013a to 0142:
  0130                                 39 30 00 00 00 00
  0140  00 00
Disassembly of executable contents:
  0x000:                               | .pos 0x000 code
  0x000: 70 18 01 00 00 00 00 00 00    |   jmp 0x118

  0x118:                               | .pos 0x118 code
  0x118:                               | _start:
  0x118: 10                            |   nop
  0x119: 00                            |   halt

==1602934== 
==1602934== HEAP SUMMARY:
==1602934==     in use at exit: 0 bytes in 0 blocks
==1602934==   total heap usage: 4 allocs, 4 frees, 20,952 bytes allocated
==1602934== 
==1602934== All heap blocks were freed -- no leaks are possible
==1602934== 
==1602934== For lists of detected and suppressed errors, rerun with: -s
==1602934== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
