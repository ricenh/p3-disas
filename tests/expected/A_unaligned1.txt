01 00 18 01 10 00 03 00  00 00 00 00 45 4c 46 00
Mini-ELF version 1
Entry point 0x118
There are 3 program headers, starting at offset 16 (0x10)
There is no symbol table present
There is no string table present
 Segment   Offset    Size      VirtAddr  Type      Flags
  00       0x004c    0x0009    0x0000    CODE      R X
  01       0x0055    0x0002    0x0118    CODE      R X
  02       0x0057    0x0008    0x013a    DATA      RW 
Contents of memory from 0000 to 0009:
  0000  70 18 01 00 00 00 00 00  00
Contents of memory from 0118 to 011a:
  0110                           10 00
Contents of memory from 013a to 0142:
  0130                                 39 30 00 00 00 00
  0140  00 00
Disassembly of executable contents:
  0x000:                               | .pos 0x000 code
  0x000: 70 18 01 00 00 00 00 00 00    |   jmp 0x118

  0x118:                               | .pos 0x118 code
  0x118:                               | _start:
  0x118: 10                            |   nop
  0x119: 00                            |   halt

